package.json

```diff
@@ -16,17 +16,57 @@
     "url": "https://github.com/ManageIQ/manageiq/issues"
   },
   "homepage": "https://github.com/ManageIQ/manageiq#readme",
+  "dependencies": {
+    "@angular/common": "~4.0.3",
+    "@angular/compiler": "~4.0.3",
+    "@angular/core": "~4.0.3",
+    "@angular/platform-browser": "~4.0.3",
+    "@angular/platform-browser-dynamic": "~4.0.3",
+    "core-js": "~2.4.1",
+    "rxjs": "~5.3.0",
+    "zone.js": "~0.8.5"
+  },
   "devDependencies": {
-    "babel-eslint": "^6.0.4",
-    "eslint": "^3.9.1",
-    "eslint-config-angular": "^0.5.0",
-    "eslint-config-standard": "^6.2.1",
-    "eslint-plugin-angular": "^1.0.1",
-    "eslint-plugin-import": "^1.9.2",
-    "eslint-plugin-jsx-a11y": "^1.5.3",
-    "eslint-plugin-promise": "^3.3.0",
-    "eslint-plugin-react": "^5.1.1",
-    "eslint-plugin-standard": "^2.0.1"
+    "autoprefixer": "~6.7.7",
+    "babel-core": "~6.24.1",
+    "babel-eslint": "~6.0.4",
+    "babel-loader": "~7.0",
+    "babel-plugin-syntax-dynamic-import": "^6.18.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-polyfill": "^6.23.0",
+    "babel-preset-env": "~1.4.0",
+    "coffee-loader": "~0.7.3",
+    "coffee-script": "~1.12.5",
+    "compression-webpack-plugin": "~0.4.0",
+    "css-loader": "~0.28.0",
+    "eslint": "~3.9.1",
+    "eslint-config-angular": "~0.5.0",
+    "eslint-config-standard": "~6.2.1",
+    "eslint-plugin-angular": "~1.0.1",
+    "eslint-plugin-import": "~1.9.2",
+    "eslint-plugin-jsx-a11y": "~1.5.3",
+    "eslint-plugin-promise": "~3.3.0",
+    "eslint-plugin-react": "~5.1.1",
+    "eslint-plugin-standard": "~2.0.1",
+    "extract-text-webpack-plugin": "~2.1.0",
+    "file-loader": "~0.11.1",
+    "glob": "~7.1.1",
+    "js-yaml": "~3.8.3",
+    "node-sass": "~4.5.2",
+    "path-complete-extname": "~0.1.0",
+    "postcss-loader": "~1.3.3",
+    "postcss-smart-import": "~0.6.11",
+    "precss": "~1.4.0",
+    "rails-erb-loader": "~5.0.0",
+    "resolve-url-loader": "^2.0.2",
+    "sass-loader": "~6.0.3",
+    "style-loader": "~0.16.1",
+    "ts-loader": "~2.0.3",
+    "typescript": "~2.2.2",
+    "webpack": "~2.4.1",
```

Comment:
Would you consider using 2.6.2 ? (as its seems the latest release of the 2.x branch)?